pkgs/development/beam-modules/mix-release.nix

```
@@ -199,14 +203,28 @@ stdenv.mkDerivation (
 
         mix compile --no-deps-check ${lib.concatStringsSep " " compileFlags}
 
+        ${lib.optionalString (escriptBinName != null) ''
+          mix do escript.build
```

```suggestion
          mix escript.build --no-deps-check
```

do is unnecessary for a single task, but more importantly mix may try and check deps while building.

---

Not skipping deps checking has not been an issue for me with the few escript builds I have done so far, but I have no objection to the flag which clearly has been applied to `compile` for good reason, so I'll get that added.

---

I actually tested with and without on an escript mixRelease we have, and skipping the deps check was required for our project. It should be non-impacting to those that don't need it, but critical for those who will. :)